nx_parallel/algorithms/centrality/betweenness.py

```diff
@@ -85,3 +88,71 @@ def _betweenness_centrality_node_subset(G, nodes, weight=None, endpoints=False):
         else:
             betweenness, delta = _accumulate_basic(betweenness, S, P, sigma, s)
     return betweenness
+
+
+@py_random_state(4)
+def edge_betweenness_centrality(
+    G, k=None, normalized=True, weight=None, seed=None, get_chunks="chunks"
+):
+    """The parallel computation is implemented by dividing the nodes into chunks and
+        computing edge betweenness centrality for each chunk concurrently.
+
+    Parameters
+    ------------
+    get_chunks : str, function (default = "chunks")
+        A function that takes in a list of all the nodes as input and returns an
+        iterable `node_chunks`. The default chunking is done by slicing the
+        `nodes` into `n` chunks, where `n` is the number of CPU cores.
+
+    networkx.edge_betweenness_centrality : https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.centrality.edge_betweenness_centrality.html
```

Comment:
In PR https://github.com/networkx/nx-parallel/pull/63, I've updated the docstring format a bit to match the sphinx guidelines. So, you would need to move this networkx link above the `Parameters` section, like this:
```.py
    """The parallel computation is implemented by dividing the nodes into chunks and
        computing edge betweenness centrality for each chunk concurrently.

    networkx.edge_betweenness_centrality : https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.centrality.edge_betweenness_centrality.html

    Parameters
    ------------
    get_chunks : str, function (default = "chunks")
        A function that takes in a list of all the nodes as input and returns an
        iterable `node_chunks`. The default chunking is done by slicing the
        `nodes` into `n` chunks, where `n` is the number of CPU cores.
    """
```

Reply:
thanks, fixed